### PR TITLE
New translation to spanish language.

### PR DESCRIPTION
**Pull request by huguei**
Following the same conventions of zonemaster-engine translation instructions.

### CODE REVIEW

**Comment by mattias-p** [comment]
Travis failed during its setup phase. I restarted the tests to see if the problem has been solved.

**Comment by matsduf** [comment]
@huguei, can you rebase this PR on latest version of develop branch and then do `make es.po`. Follow the relevant sections of https://github.com/zonemaster/zonemaster-engine/blob/develop/docs/Translation-translators.md

**Review by matsduf — APPROVED** [review]
This looks good as far as I can tell. I approve.

**Review by matsduf — CHANGES_REQUESTED** [review]
My wrong. Rebase to latest version of develop branch is still need.

**Comment by huguei** [comment]
Updated to the last develop version, and translated fuzzy messages.

File: share/es.po [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+"Al finalizar una ejecución, despliega un resumen del tiempo que le tomó "
+"responder a los servidores de nombre de la zona."
+
+msgid "Print the effective profile used in JSON format, then exit."
+msgstr "Despliega el perfil definitivo usado en formato JSON, luego termina."
+
+msgid ""
+"Source IP address used to send queries. Setting an IP address not correctly "
+"configured on a local network interface causes cryptic error messages."
+msgstr ""
```
Comment (matsduf) [inline share/es.po:153]: This not translated. Please update with at translation.

File: share/es.po [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+msgstr "Segundos "
+
+msgid "Level     "
+msgstr "Nivel     "
+
+msgid "Module       "
+msgstr "Módulo       "
+
+msgid "Testcase       "
+msgstr ""
```
Comment (matsduf) [inline share/es.po:202]: This is not translated. Please provide a translation.

**Review by matsduf — CHANGES_REQUESTED** [review]
There are two untranslated items.

**Comment by huguei** [comment]
Done. Sorry about that. I'm using poedit and although it highlights the fuzzy messages, it doesn't mark enough the empty ones.

**Review by matsduf — APPROVED** [review]
It looks good to me. I approve.

File: share/es.po [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+#~ msgstr ""
+#~ "Despliega la configuración definitiva usada en formato JSON, luego "
+#~ "termina."
+
+#~ msgid ""
+#~ "Local IP address that the test engine should try to send its requests "
+#~ "from."
+#~ msgstr ""
+#~ "Dirección IP local que debería usar el motor de pruebas como origen de "
+#~ "sus solicitudes."
```
Comment (pnax) [inline share/es.po:285]: Would it be safe to remove these lines?

Comment (matsduf) [reply share/es.po:285]: Yes. It is safe both to keep and remove. If we keep them now I will check all PO files after the v2021.2.1 release and create a clean-up PR. Is that OK?

Comment (pnax) [reply share/es.po:285]: It's fine for me to defer the removal (as long as we don't forger about it :) )

Comment (matsduf) [reply share/es.po:285]: I will not. @pnax, please approve.

**Review by pnax — APPROVED** [review]

**Pull request merged**